src/io/flutter/logging/FlutterLogConstants.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2018 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.logging;
+
+import com.intellij.execution.ui.ConsoleViewContentType;
+import com.intellij.openapi.editor.colors.TextAttributesKey;
+import com.intellij.openapi.util.Key;
+import org.jetbrains.annotations.NotNull;
+
+import static com.intellij.execution.ui.ConsoleViewContentType.registerNewConsoleViewType;
+
+public final class FlutterLogConstants {
+  private FlutterLogConstants() {
+  }
+
+  @NotNull
+  public static final TextAttributesKey NONE_OUTPUT_KEY =
+    TextAttributesKey.createTextAttributesKey("LOGCAT_NONE_OUTPUT", ConsoleViewContentType.NORMAL_OUTPUT_KEY);
+  @NotNull
+  public static final TextAttributesKey FINEST_OUTPUT_KEY =
+    TextAttributesKey.createTextAttributesKey("LOGCAT_FINEST_OUTPUT", ConsoleViewContentType.NORMAL_OUTPUT_KEY);
+  @NotNull
+  public static final TextAttributesKey FINER_OUTPUT_KEY =
+    TextAttributesKey.createTextAttributesKey("LOGCAT_FINER_OUTPUT", ConsoleViewContentType.NORMAL_OUTPUT_KEY);
+  @NotNull
+  public static final TextAttributesKey FINE_OUTPUT_KEY =
+    TextAttributesKey.createTextAttributesKey("LOGCAT_FINE_OUTPUT", ConsoleViewContentType.NORMAL_OUTPUT_KEY);
+  @NotNull
+  public static final TextAttributesKey CONFIG_OUTPUT_KEY =
+    TextAttributesKey.createTextAttributesKey("LOGCAT_CONFIG_OUTPUT", ConsoleViewContentType.NORMAL_OUTPUT_KEY);
+  @NotNull
+  public static final TextAttributesKey INFO_OUTPUT_KEY =
+    TextAttributesKey.createTextAttributesKey("LOGCAT_INFO_OUTPUT", ConsoleViewContentType.NORMAL_OUTPUT_KEY);
+  @NotNull
+  public static final TextAttributesKey WARNING_OUTPUT_KEY =
+    TextAttributesKey.createTextAttributesKey("LOGCAT_WARNING_OUTPUT", ConsoleViewContentType.NORMAL_OUTPUT_KEY);
+  @NotNull
+  public static final TextAttributesKey SEVERE_OUTPUT_KEY =
+    TextAttributesKey.createTextAttributesKey("LOGCAT_SEVERE_OUTPUT", ConsoleViewContentType.NORMAL_OUTPUT_KEY);
+  @NotNull
+  public static final TextAttributesKey SHOUT_OUTPUT_KEY =
+    TextAttributesKey.createTextAttributesKey("LOGCAT_SHOUT_OUTPUT", ConsoleViewContentType.NORMAL_OUTPUT_KEY);
+
+
+  @NotNull
+  public static final Key NONE = new Key("none.level.title");
+  @NotNull
+  public static final Key FINEST = new Key("finest.level.title");
+  @NotNull
+  public static final Key FINER = new Key("finer.level.title");
+  @NotNull
+  public static final Key FINE = new Key("fine.level.title");
+  @NotNull
+  public static final Key CONFIG = new Key("config.level.title");
+  @NotNull
+  public static final Key INFO = new Key("info.level.title");
+  @NotNull
+  public static final Key WARNING = new Key("warning.level.title");
+  @NotNull
+  public static final Key SEVERE = new Key("severe.level.title");
+  @NotNull
+  public static final Key SHOUT = new Key("shout.level.title");
+
+  static {
```

Comment:
This is really interesting.  So far, we're largely ignoring content types.  Do you know how they might come in handy?

Reply:
Ah, I thought it will be used in FlutterLogView.
But I'm not sure.

Also, do you know how to get color config from FlutterLogColorPage?
I mean, now we already have a setting page. But  I don't know how do we get color from the setting page to display in log :|

Reply:
I did some looking through the android sources and it's not obvious to me either. 🤔

I wonder if the console content types don't have something to do with it?

Anyway, leaving a `TODO` for now is fine.  If we don't get this fully hooked up before our next milestone (https://github.com/flutter/flutter-intellij/milestone/37), we can just comment out the plugin contribution.

Reply:
@pq, so should we fix all TODO in this PR or in next PR ?

Reply:
I mean for this issue: https://github.com/flutter/flutter-intellij/issues/2396

Reply:
Next PR is good.  I should get this integrated in a few hours. 🎉

Reply:
Ok, got it. I remove `WIP` status. So I will do another TODO in the next PR.

Reply:
Thanks!